dhis-2/dhis-api/src/main/java/org/hisp/dhis/datavalue/DataValueStore.java

```diff
@@ -78,63 +78,63 @@
      * @param dataElement the data element.
      */
     void deleteDataValues( DataElement dataElement );
-    
+
     /**
      * Returns a DataValue.
-     * 
-     * @param dataElement the DataElement of the DataValue.
-     * @param period the Period of the DataValue.
-     * @param source the Source of the DataValue.
-     * @param categoryOptionCombo the category option combo.
+     *
+     * @param dataElement          the DataElement of the DataValue.
+     * @param period               the Period of the DataValue.
+     * @param source               the Source of the DataValue.
+     * @param categoryOptionCombo  the category option combo.
      * @param attributeOptionCombo the attribute option combo.
      * @return the DataValue which corresponds to the given parameters, or null
-     *         if no match.
+     * if no match.
      */
-    DataValue getDataValue( DataElement dataElement, Period period, OrganisationUnit source, 
+    DataValue getDataValue( DataElement dataElement, Period period, OrganisationUnit source,
         CategoryOptionCombo categoryOptionCombo, CategoryOptionCombo attributeOptionCombo );
-    
+
     /**
      * Returns a soft deleted DataValue.
-     * 
+     *
      * @param dataValue the DataValue to use as parameters.
      * @return the DataValue which corresponds to the given parameters, or null
-     *         if no match.
+     * if no match.
      */
     DataValue getSoftDeletedDataValue( DataValue dataValue );
-    
+
     // -------------------------------------------------------------------------
     // Collections of DataValues
     // -------------------------------------------------------------------------
 
     /**
      * Returns data values for the given data export parameters.
-     * 
+     *
      * @param params the data export parameters.
      * @return a list of data values.
      */
     List<DataValue> getDataValues( DataExportParams params );
-    
+
     /**
      * Returns all DataValues.
-     * 
+     *
      * @return a list of all DataValues.
      */
     List<DataValue> getAllDataValues();
-    
+
     /**
      * Returns all DataValues for a given Source, Period, collection of
      * DataElements and CategoryOptionCombo.
-     * 
-     * @param source the Source of the DataValues.
-     * @param period the Period of the DataValues.
-     * @param dataElements the DataElements of the DataValues.
+     *
+     * @param source               the Source of the DataValues.

```

Comment:
Can we revert these insertions of whitespace after javadoc params? That is not part of the codestyle.